src/components/Footer.js

```diff
@@ -1,72 +1,97 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect, Suspense } from 'react'
+// import { useTranslation } from 'react-i18next'
+import { useTranslation } from 'gatsby-plugin-react-i18next'
 
 const quotes = [
   {
```

Comment:
@rafiiramii  great start - the data structure can be an object directly, not an array of an object. That way we can access the quotes for a language in one operation, no need to loop, i.e.

```
// this gives the current active language
const {language} = useLanguage(); // this is our custom hook that gets the current selected language
const quotesInLanguage = quotes[language]
// then you can just do the randomisation on `quotesInLanguage`

```